.github/ISSUE_TEMPLATE/-onboarding--welcome-a-new-team-member.md

```diff
@@ -7,51 +7,57 @@ assignees: ''
 
 ---
 
```

Comment:
Do we want to provide a time frame for this to be completed? We usually told our new onboarded employees (on the dev side at least) to take a week to digest information and do this card: do not even do dev contributions yet, just take the week.